hugo/static/.htaccess

```diff
@@ -34,7 +34,9 @@ RewriteRule ^([A-Za-z])([0-9][0-9])\-([0-9]{1,2})$ $1/$1$2/$1$2-$3.pdf [L,NC]
 RewriteRule ^([A-Za-z])([0-9][0-9])\-([0-9][0-9][0-9][0-9]).pdf$ $1/$1$2/$1$2-$3.pdf [L,NC]
 # Revisions and errata (e.g., P17-1069v2)
 RewriteRule ^([A-Za-z])([0-9][0-9])\-([0-9][0-9][0-9][0-9])([ve][0-9]+)$ $1/$1$2/$1$2-$3$4.pdf [L,NC]
-# Attachments (e.g., P17-1069.Poster.pdf); I (MJP) am actually not sure this is used, since attachments are hosted under attachments/
+# Attachments (e.g., P17-1069.Poster.pdf -> /anthology-files/attachments/P/P17/P17-1069.Poster.pdf)
+RewriteRule ^attachments/([A-Za-z])([0-9][0-9])\-([0-9][0-9][0-9][0-9])(\..*)?$ /anthology-files/attachments/$1/$1$2/$1$2-$3$4 [L,NC]
+# Old rule matching files with additional annotations on the canonical URL; I (MJP) am actually not sure whether every actually fires
 RewriteRule ^([A-Za-z])([0-9][0-9])\-([0-9][0-9][0-9][0-9])(\..*)?$ $1/$1$2/$1$2-$3$4 [L,NC]
```

Comment:
I guess this would fire for https://www.aclweb.org/anthology/D18-1012.Attachment.pdf -- that should never be generated on the website, but maybe legacy links that look like that exist somewhere?

Reply:
Yeah, I can see what it might match, but all such files seem to be hosted under `attachments/D/D18/D18-1012.Attachment.pdf`.

Reply:
Maybe I'll comment it out and we can see if it generates any problems...

Reply:
Well, I guess the matching URLs will generate 404s no matter if you comment it out or not :)